docs/installation.rst

```diff
@@ -4,79 +4,70 @@
 Installation
 ************
 
-Requirements
-============
+The SunPy project maintains a range of libraries that leverage the wider ecosystem of scientific Python packages for solar physics.
 
-sunpy requires Python 3.8 or higher.
+Installing a SunPy library
+==========================
 
-Installing Scientific Python and sunpy
-======================================
+If you do not currently have a working scientific Python distribution this guide will set you up with `Miniforge <https://conda-forge.org/#about>`__, which makes it easy to install and manage Python packages.
 
-sunpy is part of the wider ecosystem of scientific Python packages for solar physics.
-Therefore a working sunpy installation is more about installing the scientific Python ecosystem than sunpy itself.
+`To install the Miniforge Python distribution follow these instructions <https://github.com/conda-forge/miniforge#install>`__.
 
-If you do not currently have a working scientific Python distribution this guide will set you up with the Miniconda, which makes it easy to install and manage your scientific Python packages.
+We do not recommend that you use Anaconda, there are known conflicts with the "defaults" channel and the "conda-forge" channel that contains all of the SunPy libraries.
```

Comment:
Should we mention this at all or just drop all mentions of anaconda?

Reply:
Yeah, I don't even know what "anaconda" means...? I think heavily recommending miniconda+conda-forge channel is the way to go.